velox/functions/sparksql/JsonObjectKeys.h

```diff
@@ -51,10 +51,27 @@ struct JsonObjectKeysFunction {
     }
 
     for (auto field : jsonObject) {
+      if (isFatal(field.error())) {
+        // On-Demand only fully validates the values used and the structure
+        // leading to it.
+        return false;
+      }
       out.add_item().copy_from(std::string_view(field.unescaped_key()));
     }
     return true;
   }
+
+ private:
+  // TODO: After upgrade simdjson, we could replace with simdjson function
+  // relevant simdjson function:
+  // https://github.com/simdjson/simdjson/blob/master/include/simdjson/error-inl.h#L10-L12
```

Comment:
Please provide a reference link based on specific simdjson code version. When master code iterates, the link may reference the unexpected code block.

Reply:
This code just in master branch of simdjson.